packages/cubejs-schema-compiler/test/integration/postgres/sql-generation.test.ts

```diff
@@ -2975,6 +3057,37 @@ describe('SQL Generation', () => {
     }]
   ));
 
+  if (getEnv('nativeSqlPlanner')) {
```

Comment:
I think it would be more correct to mark the test as skipped in case of not using `nativeSqlPlanner` instead of just avoiding it all.